.github/dependabot.yml

```
@@ -0,0 +1,7 @@
+version: 2
+
+updates:
+  - package-ecosystem: "gomod"
+    directory: "/"
```

Let's also set
```suggestion
    directory: "/"
    open-pull-requests-limit: 2
 ```
so we don't get flooded in the first week

---

:+1: I had a look at the various options, so thanks for suggesting what may work well/better for you.